docs/about_sat/dependencies.md

```diff
@@ -40,7 +40,7 @@ subcommand. Each service or component is listed under the product it belongs to.
 - Kubernetes
 - S3
 
-### HPE Cray Supercomputing Compute Node Software Environment
+### User Services Software (USS)
```

Comment:
The official full name is ```HPE Cray Supercomputing User Services Software``` so do we need to reflect that here?  If so, the ```.spelling``` change will need to be reverted too.

Reply:
@deantroe, I initially thought of using the official name, but it doesn't look like we do that for any of the other products listed in this section. However, I can update this just to be safe.